docs/src/Icon.doc.js

```diff
@@ -55,6 +55,13 @@ card(
         type: `{ __path: string }`,
         description: `When using this prop, make sure that the viewbox around the SVG path is 24x24`,
       },
+      {
+        name: 'title',
+        type: 'string',
+        description:
+          'Used for hover text in place of accessibilityLabel if provided. Always localize the label.',
+        defaultValue: 'accessibilityLabel',
```

Comment:
`defaultValue` is for the default value of the prop if not passed. we should remove this since there is none